typescript/cli/src/context/strategies/chain/MultiChainResolver.ts

```diff
@@ -198,6 +201,14 @@ export class MultiChainResolver implements ChainResolver {
     );
   }
 
+  private getCosmosNativeChains(multiProvider: MultiProvider): ChainName[] {
+    const chains = multiProvider.getKnownChainNames();
+
+    return chains.filter(
+      (chain) => multiProvider.getProtocol(chain) === ProtocolType.CosmosNative,
+    );
+  }
+
   static forAgentKurtosis(): MultiChainResolver {
```

Comment:
sidenote: should probably create a ticket to nuke kurtosis references from cli/sdk